### PR TITLE
JFF-11

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: src/components/fields/SSICheckbox/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import { HIT_SLOP_DISTANCE } from '../../../@config/constants'
+import {
+  SSICheckboxSelectedContainerStyled as SelectedContainer,
+  SSICheckboxUnselectedContainerStyled as UnselectedContainer
+} from '../../../styles/components'
+
+export interface IProps {
+  onValueChange?: () => Promise<void>
+  defaultValue?: boolean
```
Comment (nklomp) [inline src/components/fields/SSICheckbox/index.tsx:12]: Maybe initialValue makes a bit more sense. That really makes clear the value is only used during init

File: src/components/views/SSISelectCredentialsTypeViewItem/index.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  // default values only
+  const { checked = false } = props
+
+  return (
+    <View>
+      <SSIFlexDirectionRowViewStyled>
+        <View>
+          <CheckBox
+            value={checked}
+            //onValueChange={(newValue) => setSelection(newValue)}
```
Comment (nklomp) [inline src/components/views/SSISelectCredentialsTypeViewItem/index.tsx]: Remove comment

File: src/components/views/SSISelectCredentialsTypeViewItem/index.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  SSIFlexDirectionRowViewStyled,
+  SSITextH3LightStyled as TitleCaption
+} from '../../../styles/styledComponents'
+import DateUtils from '../../../utils/DateUtils'
+import SSICredentialStatus from '../../labels/SSICredentialStatus'
+
+export interface Props {
+  id: string
+  title: string
+  // issuer: IIssuerSummary
```
Comment (nklomp) [inline src/components/views/SSISelectCredentialsTypeViewItem/index.tsx]: Remove comment

File: src/components/views/SSISelectCredentialsTypeViewItem/index.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+          <CheckBox
+            value={checked}
+            //onValueChange={(newValue) => setSelection(newValue)}
+            hideBox
+          />
+        </View>
+        <View>
+          <ContentTopContainer>
+            <TitleCaption>{props.title}</TitleCaption>
+            {/*<CredentialStatusCaption>*/}
```
Comment (nklomp) [inline src/components/views/SSISelectCredentialsTypeViewItem/index.tsx]: Remove commented tags

**Review by nklomp — CHANGES_REQUESTED** [review]
A small remark and some commented code that can be removed

File: src/components/fields/SSICheckbox/index.tsx (hunk 1 above)
Comment (BtencateSphereon) [reply src/components/fields/SSICheckbox/index.tsx]: changed to initialValue

File: src/components/views/SSISelectCredentialsTypeViewItem/index.tsx (hunk 2 above)
Comment (BtencateSphereon) [reply src/components/views/SSISelectCredentialsTypeViewItem/index.tsx]: Removed

File: src/components/views/SSISelectCredentialsTypeViewItem/index.tsx (hunk 4 above)
Comment (BtencateSphereon) [reply src/components/views/SSISelectCredentialsTypeViewItem/index.tsx]: Removed

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**